decred_wallet/Base.lproj/Main.storyboard

```diff
@@ -729,10 +735,10 @@
                                 </constraints>
                             </scrollView>
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="wB1-BW-PiR">
-                                <rect key="frame" x="20" y="634" width="335" height="50"/>
+                                <rect key="frame" x="20" y="525.5" width="335" height="64"/>
                                 <subviews>
                                     <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Failure to keep this seed phrase can result in theft of your funds and inability to recover your wallet." textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="3" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="T26-ed-GAn">
```

Comment:
This has a fixed number of lines. The "write down this seed..." label at the top does not. See how setting a fixed number of lines causes text to cut off on iPhone 4s and causes excess whitespace on iPad 12.9 (screenshots from Interface Builder):

_Also notice the excess spacing between the back button and the page title on the iPad_

<img width="257" alt="Screenshot 2019-05-01 at 4 06 04 AM" src="https://user-images.githubusercontent.com/18400051/57004418-cab3fa00-6bc6-11e9-8fc1-1b74a17d669a.png">
<img width="369" alt="Screenshot 2019-05-01 at 4 06 44 AM" src="https://user-images.githubusercontent.com/18400051/57004419-cab3fa00-6bc6-11e9-8511-9fb7e5006992.png">